### PR TITLE
Fix README from `linuxbsd_headers`

### PR DESCRIPTION
**Pull request by mortifiedtux**
This PR fixes formatting & spelling errors and bad/dead links in `thirdparty/linuxbsd_headers/README.md`

### CODE REVIEW

File: thirdparty/linuxbsd_headers/README.md [hunk 1]
```diff
@@ -9,20 +9,21 @@ readability.
...
 Patches in the `patches` directory should be re-applied after updates.
 
+
 ## dbus
 
-- Upstream: https://www.freedesktop.org/wiki/software/dbus/
+- Upstream: https://www.freedesktop.org/wiki/Software/dbus/
 - Version: 1.12.2
-- License: AFL-2.1
+- License: GPL-2.1
```
Comment (AThousandShips) [inline thirdparty/linuxbsd_headers/README.md:21]: The files say `Academic Free License`, if they *are* under the GPL we can't use them AFAIK, see [here](https://docs.godotengine.org/en/stable/contributing/development/best_practices_for_engine_contributors.html#don-t-use-complex-canned-solutions-for-simple-problems)

Comment (programneer) [reply thirdparty/linuxbsd_headers/README.md]: Sorry about that, fixed.

**Review by AThousandShips — APPROVED** [review]
Style and proofreading is correct, confirmed links

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**